### PR TITLE
post: happy new year by cyril

### PR DESCRIPTION
**Pull request by clacote**
PR on #164 (not on master).

* fixed typos viewed in #164
* add logos as eye candies
* mention PonyRacer and show our awesome ponies
* rewritten some text
* add a French version

### CODE REVIEW

**Comment by clacote** [comment]
We forget about PonyRacer and our awesome ponies! So I've just added that.

File: _posts/2016-01-08-new-angular-2-chapter-for-new-year.md [hunk 1]
```diff
@@ -1,27 +1,51 @@
...
+
 Maybe you haven’t heard but Angular 2 is now out in beta!
-That’s not the final release yet, but we’re getting there.
+That’s not the final release yet, but it will be there soon.
+
 And we have been hard at work on [our ebook](https://books.ninja-squad.com/angular2).
-**In fact we are just a few days shy from our first release.**
+**In fact we are just a few days shy from our first release**.
+
 Like the framework we are going to release a first version of the ebook
```
Comment (clacote) [inline _posts/2016-01-08-new-angular-2-chapter-for-new-year.md]: Comma after framework

**Comment by cexbrayat** [comment]
Last feedback in eb9d641 

Good to publish?

**Comment by cexbrayat** [comment]
@jnizet want to have a look before?

**Comment by cexbrayat** [comment]
I remove 'a few days shy' in the last commit

File: _posts/2016-01-08-new-angular-2-chapter-for-new-year.md [hunk 2]
```diff
@@ -1,27 +1,51 @@
...
+
 Maybe you haven’t heard but Angular 2 is now out in beta!
-That’s not the final release yet, but we’re getting there.
+That’s not the final release yet, but it will be there soon.
+
 And we have been hard at work on [our ebook](https://books.ninja-squad.com/angular2).
-**In fact we are just a few days shy from our first release.**
-Like the framework we are going to release a first version of the ebook
-(in english and in french) with all
+**In fact, we'll release a first version of our ebook in a few days**.
```
Comment (jnizet) [inline _posts/2016-01-08-new-angular-2-chapter-for-new-year.md]: we will

**Comment by clacote** [comment]
That's not a PR on master (unless I screwed up) : you can merge it and still have your #164 to be reviewed by @jnizet

File: _posts/2016-01-08-new-angular-2-chapter-for-new-year.md [hunk 3]
```diff
@@ -1,27 +1,51 @@
...
+Like the framework, we are going to release a first version of the ebook
+(in English and in French) with all
 that you need to write your first app,
-and then we’ll release some updates with new chapters on more advanced topics.
+and then we’ll release updates with new chapters on more advanced topics.
+Obviously, you'll be notified and will get these updates for free if you bought the book.
 
-To keep you busy while you wait for the ebook release,
-we added some content to the free samples.
+To keep you busy while you wait a few days for the ebook release,
```
Comment (jnizet) [inline _posts/2016-01-08-new-angular-2-chapter-for-new-year.md]: I'd remove "a few days". You already said it before.

File: _posts/2016-01-08-new-angular-2-chapter-for-new-year.md [hunk 4]
```diff
@@ -29,4 +53,77 @@ You can now read about :
 - The framework’s philosophy
 - How to get started
 
+Also, don't miss our sample application that leverages all what you gonna learn about Angular 2: [PonyRacer](http://ponyracer.ninja-squad.com/).
```
Comment (jnizet) [inline _posts/2016-01-08-new-angular-2-chapter-for-new-year.md]: Also, don't miss our sample application: [PonyRacer](http://ponyracer.ninja-squad.com/). It leverages all the Angular 2 features that you will learn by reading the ebook.

File: _posts/2016-01-08-new-angular-2-chapter-for-new-year.md [hunk 5]
```diff
@@ -29,4 +53,77 @@ You can now read about :
...
+
+De notre côté, nous avons travaillé dur sur [notre ebook](https://books.ninja-squad.com/angular2).
+**En fait, nous sommes enfin à quelques jours d'une première version**.
+
+Comme l'a fait l'équipe du framework, nous allons sortir une première version de l'ebook
+(en français et anglais), avec tout ce qu'il faut pour écrire ta première application,
+puis nous sortirons ensuite une mise à jour avec de nouveaux chapitres sur des sujets plus avancés.
+Tu seras bien évidemment notifié en tant qu'acheteur, et bénéficieras de ces ajouts gratuitement.
+
+Pour te faire patienter en attendant la sortie du livre dans quelques jours,
```
Comment (jnizet) [inline _posts/2016-01-08-new-angular-2-chapter-for-new-year.md]: idem: j'enlèverais "dans quelques jours"

File: _posts/2016-01-08-new-angular-2-chapter-for-new-year.md [hunk 6]
```diff
@@ -29,4 +53,77 @@ You can now read about :
...
+- TypeScript&nbsp;;
+- Les Web Components&nbsp;;
+- La philosophie du framework&nbsp;;
+- et démarrer de zéro sa première application.
+
+Aussi, ne rate pas notre application de démo qui met en œuvre tout ce que tu apprendras à faire en Angular 2&nbsp;: [PonyRacer](http://ponyracer.ninja-squad.com/).
+
+Va jeter un oeil&nbsp;!
+
+On est tellement fier de nos incroyables poneys en pixel art animé créés pour nous par le talentueux
```
Comment (jnizet) [inline _posts/2016-01-08-new-angular-2-chapter-for-new-year.md]: fiers: c'est un on "collectif"

**Comment by jnizet** [comment]
LGTM

File: _posts/2016-01-08-new-angular-2-chapter-for-new-year.md (hunk 6 above)
Comment (clacote) [inline _posts/2016-01-08-new-angular-2-chapter-for-new-year.md]: Ah, à chaque fois je me pose la question, et j'en conclue toujours à tort qu'on n'a pas le droit d'accorder au pluriel. C'est effectivement bien plus logique!

**Comment by cexbrayat** [comment]
Fixed, ready to publish.

Maybe go live when https://github.com/Ninja-Squad/ebooks/pull/101 is reviewed by one of you, and the sample reuploaded?

**Pull request merged**